stable/chartmuseum/templates/NOTES.txt

```diff
@@ -0,0 +1,3 @@
+Thank you for installing {{ .Chart.Name }}.
+
+Your release is named {{ .Release.Name }}.
```

Comment:
Can we have some additional notes on how the user can access the deployment or reach the API?